examples/fitting_NuSTAR_spectra-glesener2020.py

```diff
@@ -0,0 +1,361 @@
+"""
+=========================================================
+Fitting NuSTAR Spectra: Glesener *et al.* 2020 comparison
+=========================================================
+
+A real example from [Glesener2020]_ of fitting two NuSTAR spectra simultaneously with gain correction.
+"""
+
+import warnings
+
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.exceptions import VisibleDeprecationWarning
+from parfive import Downloader
+
+from sunkit_spex.legacy.fitting.fitter import Fitter
+
+warnings.filterwarnings("ignore", category=RuntimeWarning)
+try:
+    warnings.filterwarnings("ignore", category=VisibleDeprecationWarning)
+except AttributeError:
+    warnings.filterwarnings("ignore", category=np.exceptions.VisibleDeprecationWarning)
+
+
+#####################################################
+#
+# Set up some plotting numbers
+
+spec_single_plot_size = (8, 10)
+spec_plot_size = (25, 10)
+spec_font_size = 18
+default_font_size = 10
+x_limits, y_limits = [1.6, 8.5], [1e-1, 1e3]
+
+
+#####################################################
+#
+# # Let's fit a more realistic example
+#
+# Try fitting the spectra presented in [Glesener *et al.* 2020](https://iopscience.iop.org/article/10.3847/2041-8213/ab7341). The spectrum presented shows clear evidence of non-thermal emission in an A5.7 microflare.
+#
+# ## Let's recreate Figure 4 (left) where NuSTAR FPMB is fitted with a thermal+cold thick target model.
```

Comment:
subsection should be underlined with `----------` or bold as it is in other places